tools/fs/files.ts

```diff
@@ -187,15 +187,15 @@ export const inCheckout = _.once(function () {
 });
 
 // True if we are using a warehouse: either installed Meteor, or if
-// $METEOR_WAREHOUSE_DIR is set. Otherwise false (we're in a git checkout and
+// $METEOR_USES_WAREHOUSE is set. Otherwise false (we're in a git checkout and
 // just using packages from the checkout).
 export function usesWarehouse() {
   // Test hook: act like we're "installed" using a non-homedir warehouse
   // directory.
-  if (process.env.METEOR_WAREHOUSE_DIR) {
+  if (process.env.METEOR_USES_WAREHOUSE) {
```

Comment:
It is not clear from this change alone, it seems that `METEOR_USES_WAREHOUSE` is set exactly when `METEOR_WAREHOUSE_DIR` itself is also set. So what is the difference then?

Reply:
We set METEOR_WAREHOUSE_DIR from our scripts to override the path.